handbook/engineering/code-intelligence/index.md

```diff
@@ -38,16 +38,16 @@ Build a code intelligence platform that understands code in all languages and pr
 
 This list will be updated regularly with links to the technical implementation details in RFCs and/or Github issues.
 
-### Provide precise code intel for the primary programming languages that combined cover > 50% of usage across customer projects.
+### Provide precise code intel for the primary programming languages that combined cover > 50% of usage across organization projects.
```

Comment:
Is this 50% coverage within a single organization, or 50% coverage across all our customers/prospects?

Reply:
That's a good question! I think about it as 50% coverage across all our customers/prospects, but I'm not sure. @aidaeology can you confirm?